docs/analysing-compilation-output.rst

```diff
@@ -23,7 +23,7 @@ Consider the following contract (named, say ``contract.sol``):
 
 The following would be the output of ``solc --asm contract.sol``
 
-.. code-block:: none
+.. code-block:: assembly
```

Comment:
I think this should rather be "plain text", I don't think "assembly" highlights properly.

Reply:
Ah, just saw the comments above. I'll have a look at the preview to check if it looks nice.

Reply:
My point of view for choosing `.. code-block:: solidity` rather than using `.. code-block:: yul` is to have the code highlited in Github, since using `yul` won't highlight the code.

Below is the difference between using both: 
- `.. code-block:: solidity`
![pic-selected-210608-0736-26](https://user-images.githubusercontent.com/36774944/121129310-0d783f80-c82d-11eb-8b9d-1acdb081710f.png)

- `.. code-block:: yul`
![pic-selected-210608-0736-47](https://user-images.githubusercontent.com/36774944/121129335-1537e400-c82d-11eb-8d0d-958e6978605f.png)

Reply:
I'm not sure this is good highlighting. Colors are a bit random. Might mislead some users into thinking that some things have special meaning when they do not. My vote would be for plain text for this.

Reply:
I agree with you @cameel that might mislead some users, I will switch to `.. code-block:: none`